xbmc/utils/URIUtils.cpp

```diff
@@ -215,6 +215,17 @@ void URIUtils::Split(const std::string& strFileNameAndPath,
   strPath = strFileNameAndPath.substr(0, i+1);
   // everything to the right of the directory separator
   strFileName = strFileNameAndPath.substr(i+1);
+
+  // ignore options
```

Comment:
You might want to pick this test https://github.com/xbmc/xbmc/pull/10134/files#diff-c2c11e94617bec636884b0ccdf9dbaf6R125